packages/ui/src/Shared/NavLinkTab.css

```diff
@@ -0,0 +1,16 @@
+.NavLinkTab {
+  color: black;
+  font-size: 14px;
+}
+
+.NavLinkTab_disabled {
```

Comment:
Modifiers should be expressed like this: `.NavLinkTab.is-disabled`

Reply:
Changed name in css file as you recommend.